specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-12-01/examples/GetOfferAdminApprovalRequests.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "privateStoreId": "a0e28e55-90c4-41d8-8e34-bb7ef7775406",
+    "offerId": "marketplacetestthirdparty.md-test-third-party-2",
+    "publisherId": "marketplacetestthirdparty",
+    "api-version": "2020-12-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Marketplace/privateStores/9afd3c45-5230-4d58-9469-2cacc00bba68/requestApprovals/marketplacetestthirdparty.md-test-third-party-2",
```

Comment:
This example is for the adminRequestApprovals resource type but is using the requestApprovals resource type in the ID and Name.  If they are the same thing they shouldn't be two different resource types

Reply:
changed to support the new resources